ci/test/cargo-test-miri.sh

```diff
@@ -16,14 +16,7 @@ set -euo pipefail
 
 # miri artifacts are thoroughly incompatible with normal build artifacts,
 # so keep them away from the `target` directory.
-export CARGO_TARGET_DIR=miri-target
-
-# At the moment only ore and repr have tests meant to be run under miri.
-pkgs=(
-    repr
-    ore
-)
-
-for pkg in "${pkgs[@]}"; do
-    (cd src/"$pkg" && cargo miri test miri --all-features)
-done
+export CARGO_TARGET_DIR="$PWD/miri-target"
+export MIRIFLAGS="-Zmiri-disable-isolation"
```

Comment:
It would also be nice to enable `-Zmiri-strict-provenance`, if it doesn't cause too many more tests to fail

Reply:
Seems to work fine, enabling.

Reply:
Great!